.github/workflows/verify.yml

```diff
@@ -9,14 +9,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Setup Node 21.x
+      - name: Setup Node 22
         uses: actions/setup-node@v4
         with:
-          node-version: 21.x
+          node-version: 22
           cache: 'npm'
 
       - name: Install Dependencies
         run: npm ci
 
       - name: Linting & Formatting
         run: npm run lint
+
+      - name: Test
+        run: npm test
```

Comment:
Adds some simple tests just for the resolver implementation, to make sure it’s not completely borked